simphony/cuds/tests/test_mesh.py

```diff
@@ -680,5 +682,48 @@ def test_update_cell(self):
 
         self.assertItemsEqual(cell_upd.points, cell_ret.points)
 
+    def test_set_data(self):
+        """ Check that data can be retrieved
+
+        """
+
+        org_data = DataContainer()
+
+        org_data[CUBA.VELOCITY] = (0, 0, 0)
+
+        self.mesh.data = org_data
+        ret_data = self.mesh.data
+
+        self.assertItemsEqual(org_data, ret_data)
```

Comment:
This checks only the ``keys`` of mappings, please use the compare_data_container function as in https://github.com/simphony/simphony-common/blob/master/simphony/testing/abc_container_check.py#L15 
This will allow one to do
```
self.assertEqual(org_data, ret_data)
```